app/javascript/components/conditions/ConditionSetFormField/ConditionFormField/model.js

```diff
@@ -53,6 +53,35 @@ class ConditionModel {
   @observable
   remove;
 
+  /**
+   * Return a string representation of the current value.
+   * Both user-facing and sent to the backend when performing a search.
+   */
+  @computed
+  get currTextValue() {
+    if (this.optionSetId) {
+      const lastIndex = this.levels.length - 1;
+
+      // Iterate back through each level, trying to find the deepest selected item.
+      for (let i = lastIndex; i >= 0; i -= 1) {
+        const { selected, options } = this.levels[i];
+
+        if (selected != null) {
+          const { name = '' } = options.find(({ id }) => selected === id) || {};
+          return name;
+        }
+      }
+
+      return null;
+    } if (this.optionNodeId) {
```

Comment:
Is this supposed to be `else if`?

Reply:
There's a [lint check](https://eslint.org/docs/rules/no-else-return) that `else` after a `return` doesn't make sense because the else is implicit. Apparently airbnb explicitly changed the default to also prevent `else if`. But I don't like that rule, I'll turn it off and fix this.